index.html

```diff
@@ -6,6 +6,7 @@
 
 <link rel="stylesheet" href="./local.css" />
 <link rel="stylesheet" href="./print.css" />
+<link rel="stylesheet" href="./dark.css" />
```

Comment:
Since we're adding a new file, please update https://github.com/w3c/clreq/blob/gh-pages/echidna too.